crud.py

```diff
@@ -0,0 +1,67 @@
+from sqlalchemy.orm import Session
+
+from db import models, schemas
+from db.models import DBAuthor, DBBook
+
+
+def get_author_by_name(
+    db: Session,
+    name: str,
+) -> DBAuthor | None:
+    return db.query(models.DBAuthor).filter(models.DBAuthor.name == name).first()
```

Comment:
In this case in all places instead of models.DBAuthor you can use DBAuthor, the same with DBBook